README.md

```diff
@@ -1 +1,15 @@
-# hello-microverse-projec
\ No newline at end of file
+Hello Microverse Project
+
+In this project, I'm just setting a repository to show my Github workflow best practices.
+
+👤 Author1
+
+GitHub: @paulinagonzalez
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] `required`
- [ ] Please consider writting a more descriptive `readme` including social handle like `twitter` & `linkedin` so that recruiters can easily navigate you. please follow this [template](https://github.com/microverseinc/readme-template)